spec/factories/notification_phone_configurations.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  Faker::Config.locale = :en
+
+  factory :notification_phone_configuration do
+    phone { '+1 202-555-1212' }
+    in_person_enrollment { association :in_person_enrollment }
+  end
+
+  trait :notifcation_sent do
```

Comment:
change `notifcation_sent` to `notification _sent`

Reply:
Good 👀 ! I ended up removing this as part of another comment, in 08db9e59798f313831ff2f6eba07937877a42e1b